src/cli/Cargo.toml

```diff
@@ -24,7 +24,6 @@ cluster_components = ["fluvio-spu", "fluvio-sc"]
 rustc_version = "0.2.3"
 
```

Comment:
update version for cli

Reply:
What version should this be updated to? And do we need to update `rustc_version` ?

Reply:
`cluster` crate should be bump to `0.3.0` if there are API incompatibilities, otherwise jump bump patch number.  

`rustc` should be update to: `rustc 1.47.0`